app/src/main/java/tv/vizbee/demo/model/LoginRequest.kt

```diff
@@ -0,0 +1,7 @@
+package tv.vizbee.demo.model
+
+data class LoginRequest(
+    val email: String,
+    val pwd: String,
+    val deviceId: String
```

Comment:
you can use default value for the deviceId since its constant through out

Reply:
Done